syntax = "proto3";

package KBchulan.ChatRoom.StatusServer;

// 获取tcp服务器的请求
message GetTcpServerRequest
{
  string uuid = 1;
}

// 获取tcp服务器的响应
message GetTcpServerResponse
{
  int32 code = 1;
  string message = 2;
  map<string, string> data = 3;
}

// 获取tcp服务器的接口
service StatusService
{
  // 获取tcp服务器的 RPC 方法
  rpc GetTcpServer(GetTcpServerRequest) returns (GetTcpServerResponse);
}